applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_integration_coefficient_calculator.cpp

```diff
@@ -0,0 +1,54 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//                   Gennady Markelov
+//
+
+#include "custom_elements/integration_coefficients_calculator.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+
+#include <boost/numeric/ublas/assignment.hpp>
+
+using namespace Kratos;
+using namespace std::string_literals;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(IntegrationCoefficientsCalculator_ReturnsCorrectValue, KratosGeoMechanicsFastSuiteWithoutKernel)
```

Comment:
Perhaps we should be more explicit about these tests using a calculator without a modifier, e.g.:
```suggestion
KRATOS_TEST_CASE_IN_SUITE(IntegrationCoefficientsCalculatorWithoutModifier_ReturnsCorrectValue, KratosGeoMechanicsFastSuiteWithoutKernel)
```

This suggestion also applies to the next test case.

Reply:
renamed